pkg/feature/storage/evaluation_event.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 The Bucketeer Authors.
```

Comment:
The evaluation events are managed by the event counter service, so this file should be under the `/pkg/eventcounter/storage/v2/` package.